src/components/RightSidebar/Description/Description.vue

```diff
@@ -242,8 +242,8 @@ export default {
 			if (!this.canSubmit) {
 				return
 			}
-			// Remove newlines and whitespaces.
-			this.descriptionText = this.descriptionText.replace(/\r\n|\n|\r/gm, '').trim()
+			// Remove leading/trailing whitespaces.
+			this.descriptionText = this.descriptionText.replace(/\r\n|\n|\r/gm, '\n').trim()
```

Comment:
Shouldn't this be in the component directly?

Reply:
not sure, does it make sense to allow those others to be posted, thinking about gist-alike usecases for source code where it might matter?

but would also be fine for me